source/dofs/dof_tools_constraints.cc

```diff
@@ -2659,9 +2667,9 @@ namespace DoFTools
                         weights[wi][wj] = copy_data.global_parameter_representation[pos](i);
                       }
                   }
-                else
-                  Assert (copy_data.global_parameter_representation[pos](i) == 0,
-                          ExcInternalError());
+              //else
+              //  Assert (copy_data.global_parameter_representation[pos](i) == 0,
+              //          ExcInternalError());
```

Comment:
In parallel case this assertation does not always hold. This is legitimate as far as I understand. Maybe a better idea would be to soround it with #ifndef DEAL_II_WITH_MPI?